app/models/user.rb

```diff
@@ -1,5 +1,5 @@
 class User < ApplicationRecord
-  self.ignored_columns = %w[encrypted_phone totp_timestamp]
+  self.ignored_columns = %w[totp_timestamp]
```

Comment:
can we also drop this 😬

Reply:
you mean drop the `totp_timestamp` column?

Reply:
Yeah, it's been ignored for...[2.5 years](https://github.com/18F/identity-idp/pull/3524) so I think we're good 😀